components/workflow.rst

```diff
@@ -66,11 +68,11 @@ are trying to use it with::
     use Symfony\Component\Workflow\Registry;
     use Symfony\Component\Workflow\SupportStrategy\InstanceOfSupportStrategy;
 
-    $blogWorkflow = ...
+    $blogPostWorkflow = ...
```

Comment:
I propose to rename it to “Article” instead of “BlogPost”, it’s much shorter

Reply:
@OskarStark so should i change also all the wording related to blog or blogpost in this doc page to article ?

Reply:
@javiereguiluz what do you think?

Reply:
@javiereguiluz what do you think?

Reply:
I like `blogPost` because it's what we traditionally used in the Workflow article. "article" is nice too, but I wouldn't mind keep using "blogPost". Thanks.